.github/workflows/fcos-podman-next-build.yml

```diff
@@ -17,36 +19,66 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - name: Install qemu dependency
+      run: |
+        sudo apt update
+        sudo apt -y install qemu-user-static
+
     - name: Set up wait-for-copr
       run: |
         pip3 install git+https://github.com/packit/wait-for-copr.git@main
 
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Get short SHA from HEAD
+      run: echo "SHORT_SHA=$(git rev-parse --short HEAD)" >> "$GITHUB_ENV"
+      id: short_sha
+
     - name: Wait for successful podman-next build with the latest commit
       run: |
         # TODO: add this in the Containerfile itself or as a --build-arg
-        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman $(git rev-parse --short ${{ github.sha }})
+        wait-for-copr --owner ${{ env.COPR_OWNER }} --project ${{ env.COPR_PROJECT }} podman $SHORT_SHA
         echo "podman-next build successful."
 
-    - name: Check out code
-      uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
-
     - name: Build FCOS Image
-      id: build_image
+      id: build_image_multiarch
       # Ref: https://github.com/redhat-actions/buildah-build
       uses: redhat-actions/buildah-build@v2
       with:
-        image: ${{ env.IMAGE_REGISTRY }}/${{ env.IMAGE_NAME }}
-        tags: ${{env.IMAGE_TAGS }}
-        containerfiles: |
-          ./contrib/podman-next/fcos-podmanimage/Containerfile .
+        image: ${{ env.IMAGE_NAME }}
+        tags: ${{ env.IMAGE_TAGS }}
```

Comment:
Minor: Tagging the image with a full sha still tickles my funny-bone slightly.  Mostly because in quay (or podman pull), it's not entirely obvious what the sha pertains to.  For example, it could be a digest from another image, a hash of some file, etc.  Would you consider using a prefixed tag instead, maybe something like `podman-${{ env.SHORTSHA }}`?  

Even better would be to (somehow) grab the V-R from the RPM's NVR and use that as the tag (along with `env.SHORTSHA`).

Reply:
Getting version-release in github action is probably convoluted. I'm gonna leave that for later. But `podman-shortsha` would be doable. @benoitf would you be ok with `podman-SHORTSHA` as a quay tag instead of the full sha ?

Reply:
sure short sha works `podman-shortsha` or `git-shortsha` as well

Reply:
the github action now says `podman-SHORTSHA`. A sample image is already available at https://quay.io/repository/podman/fcos?tab=tags in that format.